core/src/main/scala/com/nvidia/spark/rapids/tool/profiling/ProfileArgs.scala

```diff
@@ -102,6 +102,11 @@ Usage: java -cp rapids-4-spark-tools_2.12-<version>.jar:$SPARK_HOME/jars/*
     opt[Boolean](required = false,
       descr = "Toggle AutoTuner module.",
       default = Some(false))
+  val outputSqlIdsAligned: ScallopOption[Boolean] =
```

Comment:
Do we need a new argument? If we do and the new feature does not have much impact on performance, then should we keep the default to be true ?